bridger-agent/src/components/TransactionTable/TransactionTable.tsx

```diff
@@ -0,0 +1,243 @@
+"use client";
+
+import { useState } from "react";
+import { Button } from "@chakra-ui/react";
+import { Transaction, FilterableDisplayElement } from "./types";
+import { FilterableDropdown } from "./FilterableDropdown";
+
+const dummyTransactions: Transaction[] = [
```

Comment:
Note from reviewer: these would be replaced with actual data-loading in the future